src/Client.php

```diff
@@ -245,7 +245,7 @@ public function getConnectPassword(): string
      */
     public function getConnectPort(): string
     {
-        return $this->connectPort;
+        return strval($this->connectPort);
```

Comment:
This change is required to not introduce breaking changes. Method will return string as it was before.